torch/_inductor/scheduler.py

```diff
@@ -806,42 +817,51 @@ def fuse_nodes_once(self):
             - self.score_fusion(): assigns priority to a given fusion
         """
         fused_nodes = set(self.nodes)
-        for node1, node2 in self.get_possible_fusions():
-            node1 = self.name_to_fused_node[node1.get_first_name()]
-            node2 = self.name_to_fused_node[node2.get_first_name()]
+
+        recent_fusions = set()
```

Comment:
This is required because `get_possible_fusions()` may return a new node not in `name_to_fused_node`, so we can't trust
```
node = self.name_to_fused_node[node.get_first_name()]
```
to map an unfused nodes back to itself